Readme.md

```diff
@@ -1,21 +1,146 @@
+    - **DAO and Plugin Registries**: registers into our protocol those DAOs or plugins.
+    - **Plugin Processor:** installs and uninstalls plugins into DAOs.
+
+Additionally to those two sections, we have developed several plugins DAOs can easily install upon creation. These are:
+
+- __Token Voting plugin__: enabling voting through token majority as a governance mechanism for DAOs
+- __Multisig plugin__: enabling assets to be transacted based on approval from members.
+- __Admin plugin__: enabling one account to have god-like access to the DAO.
+- __Addresslist Voting plugin__: a set of pre-selected addresses are able to vote within a DAO.
+
+Let's dive into more detail on each of these sections.
+
+### Core Contracts
+
+The *Core Contracts* describe how every DAO generated in the Aragon OSx protocol will be set up. It is extremely lean and by design and constitutes the most critical aspects of our architecture.
+
+In a nutshell, each DAO is composed of 3 interconnecting components:
+
+1. **The DAO treasury:** The DAO contract is where the **core functionality** of the DAO lies. It is the contract in charge of:
+    - Representing the identity of the DAO (ENS name, logo, description, other metadata)
+    - Holding and managing the treasury
+    - Executing arbitrary actions to:
+        - transfer assets
+        - call its own functions
+        - call functions in external contracts
+    - Providing general technical utilities (signature validation, callback handling)
+2. **The Permission Manager:** The permission manager is an integral part of any DAO and the center of our protocol architecture. It **manages permissions for the DAO** by specifying which addresses have permission to call distinct functions on contracts associated with your DAO.
+3. **Plugin Manager**: Any custom functionality can be added or removed through plugins, allowing you to **fully customize your DAO**. Some examples of plugins DAOs could install are:
+    - Governance (e.g., token voting, one-person one-vote)
+    - Asset management (e.g., ERC-20 or NFT minting, token streaming, DeFi)
+    - Membership (governing budget allowances, gating access, curating a member list)
+
+The following graphic shows how an exemplary DAO setup, where the
+
+![An examplary DAO setup](https://devs.aragon.org/assets/images/dao-plugin.drawio-7086d0911d25218097dae94665b1a7b1.svg)
+
+An examplary DAO setup showing interactions between the three core contract pieces triggered by different user groups: The `DAO` and `PermissionManager` contract in blue and red, respectively, as well as two `Plugin` contracts in green. Function calls are visualized as black arrows and require permission checks (red, dashed arrow). In this example, the permission manager determines whether the token voting plugin can execute actions on the DAO, a member can change its settings, or if an DeFi-related plugin is allowed to invest in a certain, external contract.
```

Comment:
Just to clarify, the DAO and the PermissionManager are shipped as one contract that does both. This might lead to confusion, I would emphasize that they are the 2 components within the DAO contract